codice/ddf
# DDF-4373 Add support for search forms with point radii

#### What does this PR do?
Enable users to create and use search forms with point radii
#### Who is reviewing it? 

@GabrielFabian 	@gjvera 	@willwill96 	@rymach 	
#### Select relevant component teams: 

#### Ask 2 committers to review/merge the PR and tag them here.

@bdeining
@djblue
#### How should this be tested?

Create search forms with point radii. Verify they save successfully and their values are correct when they are editted and used to search.
#### Any background context you want to provide?
#### What are the relevant tickets?
[DDF-4373](https://codice.atlassian.net/browse/DDF-4373)
#### Screenshots

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
